cmd/ctr/commands/images/mounts.go

```diff
@@ -0,0 +1,80 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package images
+
+import (
+	"github.com/containerd/containerd"
+	"github.com/containerd/containerd/cmd/ctr/commands"
+	"github.com/containerd/containerd/errdefs"
+	"github.com/containerd/containerd/pkg/cri/util"
+	"github.com/containerd/containerd/platforms"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+var mountsCommand = cli.Command{
+	Name:      "mounts",
+	Usage:     "prints snapshotter mounts for a snapshot of `image`",
+	ArgsUsage: "[flags] <image>",
+	Flags: append(commands.SnapshotterFlags,
+		cli.StringFlag{
+			Name:   "platform",
+			Usage:  "platform to create snapshot for",
+			EnvVar: "CONTAINERD_PLATFORM",
+		},
+	),
+	Action: func(context *cli.Context) (retErr error) {
+		ref := context.Args().First()
+		client, ctx, cancel, err := commands.NewClient(context)
+		if err != nil {
+			return err
+		}
+		defer cancel()
+
+		snapshotter := context.String("snapshotter")
+		if snapshotter == "" {
+			snapshotter = containerd.DefaultSnapshotter
+		}
+
```

Comment:
Don't need this, you can just pass `context.String("snapshotter")` to `client.SnapshotService(snapshotter)` as it will return the default snapshotter if it can't find the one requested. Don't know of any snapshotter that's registered for empty string (I hope there isn't one 😄)